inference-engine/src/transformations/include/transformations/serialize.hpp

```diff
@@ -32,17 +31,25 @@ class TRANSFORMATIONS_API StreamSerialize;
  */
 class ngraph::pass::Serialize : public ngraph::pass::FunctionPass {
 public:
-    enum class Version { IR_V10 };
+    enum class Version : uint8_t {
+        UNSPECIFIED = 0,        // Use version from the function
+        IR_V10 = 10,            // v10 IR
+        IR_V11 = 11             // v11 IR
+    };
     NGRAPH_RTTI_DECLARATION;
     bool run_on_function(std::shared_ptr<ngraph::Function> f) override;
 
-    Serialize(std::ostream & xmlFile, std::ostream & binFile,
-              Version version = Version::IR_V10,
-              std::map<std::string, ngraph::OpSet> custom_opsets = {});
+    Serialize(std::ostream& xmlFile, std::ostream& binFile,
+              std::map<std::string, ngraph::OpSet> custom_opsets,
+              Version version = Version::UNSPECIFIED);
+    Serialize(std::ostream& xmlFile, std::ostream& binFile,
+              Version version = Version::UNSPECIFIED);
 
     Serialize(const std::string& xmlPath, const std::string& binPath,
```

Comment:
will we deprecated ctor with custom_opsets in new API?
Looks like end customer have to use new RTTI and don't use custom opsets for serialize

Reply:
I propose to do it in the next PR with public transformation